lore/end_users/status/README.md

```diff
@@ -8,16 +8,21 @@ permalink: /lore/end_users/status
 
 ## Table of Contents
 
+* [Zuul Status](#zuul-status)
 * [BonnyCI Status ScreenBoard](#bonnyci-status-screenboard)
   * [System Load](#system-load)
   * [Disk Free](#disk-free)
   * [Zuul Event Queue Length](#zuul-event-queue-length)
   * [Zuul Pipelines](#zuul-pipelines)
   * [Service Status](#service-status)
 
+## Zuul Status
+
+The [Zuul Status page](http://zuul.bonnyci.org/) has information on queue length and pipelines. When code is being tested it is listed under the pipeline and provides links to more information about them.
```

Comment:
This sentence is a little confusing. Maybe something like "Code currently being tested in the pipeline is listed under the pipeline heading. This listing also contains links for more detailed information on (status... etc... say what exactly here)."